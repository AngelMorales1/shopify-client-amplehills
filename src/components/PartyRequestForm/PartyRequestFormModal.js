import React from 'react';
import cx from 'classnames';

import styles from './PartyRequestForm.scss';
import { Button } from 'components/base';

const PartyRequestFormModal = ({ children, onCloseClick, title }) => {
  return (
    <div
      className={cx(
        'overflow-scroll fixed-cover bg-white-wash flex justify-center items-center transition-fade-in'
      )}
    >
      <div
        className={cx(
          'wh100 modal modal--light-gray-border max-width-2 mx-auto relative flex items-center justify-center bg-white drop-shadow transition-slide-up-large-long'
        )}
      >
        <div className="wh100 m-auto flex flex-column mb3 justify-center">
          <div
            className={cx(
              styles['PartyRequestForm__modal__info'],
              'absolute t0 l0 w100 p3 overflow-scroll'
            )}
          >
            <h3 className="callout mb3">{title}</h3>
            {children}
          </div>
          <div
            className={cx(
              styles['PartyRequestForm__modal__close-bar'],
              'absolute b0 l0 w100 p1 bg-white flex flex-row justify-end items-center'
            )}
          >
            <Button
              color="madison-blue"
              variant="primary-small"
              label="Close"
              onClick={onCloseClick}
            />
          </div>
        </div>
      </div>
    </div>
  );
};

export default PartyRequestFormModal;
