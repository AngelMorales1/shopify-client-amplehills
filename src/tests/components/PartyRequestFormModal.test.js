import React from 'react';
import { shallow } from 'enzyme';

import PartyRequestFormModal from 'components/PartyRequestForm/PartyRequestFormModal';

it('renders without data', () => {
  const component = shallow(<PartyRequestFormModal />);

  expect(component).toMatchSnapshot();
});

it('renders with data', () => {
  const component = shallow(
    <PartyRequestFormModal onCloseClick={() => console.log('foo')} />
  );

  expect(component).toMatchSnapshot();
});
