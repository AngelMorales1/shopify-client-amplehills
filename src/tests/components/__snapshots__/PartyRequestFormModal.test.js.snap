// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with data 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PartyRequestFormModal
    onCloseClick={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="wh100 modal modal--light-gray-border max-width-2 mx-auto relative flex items-center justify-center bg-white drop-shadow transition-slide-up-large-long"
>
        <div
                className="wh100 m-auto flex flex-column mb3 justify-center"
        >
                <div
                        className="absolute t0 l0 w100 p3 overflow-scroll"
                >
                        <h3
                                className="callout mb3"
                        />
                </div>
                <div
                        className="absolute b0 l0 w100 p2 bg-white flex flex-row justify-end items-center"
                >
                        <Button
                                childrenWrapperClassName=""
                                className=""
                                color="madison-blue"
                                disabled={false}
                                fullWidth={false}
                                hover="shadow"
                                label="Close"
                                onClick={[Function]}
                                onMouseEnter={[Function]}
                                onMouseLeave={[Function]}
                                openInCurrentTab={false}
                                shadow={false}
                                to=""
                                type="button"
                                variant="primary-small"
                        />
                </div>
        </div>
</div>,
      "className": "overflow-scroll fixed-cover bg-white-wash flex justify-center items-center transition-fade-in",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="wh100 m-auto flex flex-column mb3 justify-center"
>
          <div
                    className="absolute t0 l0 w100 p3 overflow-scroll"
          >
                    <h3
                              className="callout mb3"
                    />
          </div>
          <div
                    className="absolute b0 l0 w100 p2 bg-white flex flex-row justify-end items-center"
          >
                    <Button
                              childrenWrapperClassName=""
                              className=""
                              color="madison-blue"
                              disabled={false}
                              fullWidth={false}
                              hover="shadow"
                              label="Close"
                              onClick={[Function]}
                              onMouseEnter={[Function]}
                              onMouseLeave={[Function]}
                              openInCurrentTab={false}
                              shadow={false}
                              to=""
                              type="button"
                              variant="primary-small"
                    />
          </div>
</div>,
        "className": "wh100 modal modal--light-gray-border max-width-2 mx-auto relative flex items-center justify-center bg-white drop-shadow transition-slide-up-large-long",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="absolute t0 l0 w100 p3 overflow-scroll"
>
              <h3
                            className="callout mb3"
              />
</div>,
            <div
              className="absolute b0 l0 w100 p2 bg-white flex flex-row justify-end items-center"
>
              <Button
                            childrenWrapperClassName=""
                            className=""
                            color="madison-blue"
                            disabled={false}
                            fullWidth={false}
                            hover="shadow"
                            label="Close"
                            onClick={[Function]}
                            onMouseEnter={[Function]}
                            onMouseLeave={[Function]}
                            openInCurrentTab={false}
                            shadow={false}
                            to=""
                            type="button"
                            variant="primary-small"
              />
</div>,
          ],
          "className": "wh100 m-auto flex flex-column mb3 justify-center",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3
                  className="callout mb3"
/>,
                undefined,
              ],
              "className": "absolute t0 l0 w100 p3 overflow-scroll",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": undefined,
                  "className": "callout mb3",
                },
                "ref": null,
                "rendered": null,
                "type": "h3",
              },
              undefined,
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Button
                childrenWrapperClassName=""
                className=""
                color="madison-blue"
                disabled={false}
                fullWidth={false}
                hover="shadow"
                label="Close"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onMouseLeave={[Function]}
                openInCurrentTab={false}
                shadow={false}
                to=""
                type="button"
                variant="primary-small"
/>,
              "className": "absolute b0 l0 w100 p2 bg-white flex flex-row justify-end items-center",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": null,
                "childrenWrapperClassName": "",
                "className": "",
                "color": "madison-blue",
                "disabled": false,
                "fullWidth": false,
                "hover": "shadow",
                "label": "Close",
                "onClick": [Function],
                "onMouseEnter": [Function],
                "onMouseLeave": [Function],
                "openInCurrentTab": false,
                "shadow": false,
                "to": "",
                "type": "button",
                "variant": "primary-small",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="wh100 modal modal--light-gray-border max-width-2 mx-auto relative flex items-center justify-center bg-white drop-shadow transition-slide-up-large-long"
>
          <div
                    className="wh100 m-auto flex flex-column mb3 justify-center"
          >
                    <div
                              className="absolute t0 l0 w100 p3 overflow-scroll"
                    >
                              <h3
                                        className="callout mb3"
                              />
                    </div>
                    <div
                              className="absolute b0 l0 w100 p2 bg-white flex flex-row justify-end items-center"
                    >
                              <Button
                                        childrenWrapperClassName=""
                                        className=""
                                        color="madison-blue"
                                        disabled={false}
                                        fullWidth={false}
                                        hover="shadow"
                                        label="Close"
                                        onClick={[Function]}
                                        onMouseEnter={[Function]}
                                        onMouseLeave={[Function]}
                                        openInCurrentTab={false}
                                        shadow={false}
                                        to=""
                                        type="button"
                                        variant="primary-small"
                              />
                    </div>
          </div>
</div>,
        "className": "overflow-scroll fixed-cover bg-white-wash flex justify-center items-center transition-fade-in",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="wh100 m-auto flex flex-column mb3 justify-center"
>
            <div
                        className="absolute t0 l0 w100 p3 overflow-scroll"
            >
                        <h3
                                    className="callout mb3"
                        />
            </div>
            <div
                        className="absolute b0 l0 w100 p2 bg-white flex flex-row justify-end items-center"
            >
                        <Button
                                    childrenWrapperClassName=""
                                    className=""
                                    color="madison-blue"
                                    disabled={false}
                                    fullWidth={false}
                                    hover="shadow"
                                    label="Close"
                                    onClick={[Function]}
                                    onMouseEnter={[Function]}
                                    onMouseLeave={[Function]}
                                    openInCurrentTab={false}
                                    shadow={false}
                                    to=""
                                    type="button"
                                    variant="primary-small"
                        />
            </div>
</div>,
          "className": "wh100 modal modal--light-gray-border max-width-2 mx-auto relative flex items-center justify-center bg-white drop-shadow transition-slide-up-large-long",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="absolute t0 l0 w100 p3 overflow-scroll"
>
                <h3
                                className="callout mb3"
                />
</div>,
              <div
                className="absolute b0 l0 w100 p2 bg-white flex flex-row justify-end items-center"
>
                <Button
                                childrenWrapperClassName=""
                                className=""
                                color="madison-blue"
                                disabled={false}
                                fullWidth={false}
                                hover="shadow"
                                label="Close"
                                onClick={[Function]}
                                onMouseEnter={[Function]}
                                onMouseLeave={[Function]}
                                openInCurrentTab={false}
                                shadow={false}
                                to=""
                                type="button"
                                variant="primary-small"
                />
</div>,
            ],
            "className": "wh100 m-auto flex flex-column mb3 justify-center",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3
                    className="callout mb3"
/>,
                  undefined,
                ],
                "className": "absolute t0 l0 w100 p3 overflow-scroll",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": undefined,
                    "className": "callout mb3",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "h3",
                },
                undefined,
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Button
                  childrenWrapperClassName=""
                  className=""
                  color="madison-blue"
                  disabled={false}
                  fullWidth={false}
                  hover="shadow"
                  label="Close"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  openInCurrentTab={false}
                  shadow={false}
                  to=""
                  type="button"
                  variant="primary-small"
/>,
                "className": "absolute b0 l0 w100 p2 bg-white flex flex-row justify-end items-center",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": null,
                  "childrenWrapperClassName": "",
                  "className": "",
                  "color": "madison-blue",
                  "disabled": false,
                  "fullWidth": false,
                  "hover": "shadow",
                  "label": "Close",
                  "onClick": [Function],
                  "onMouseEnter": [Function],
                  "onMouseLeave": [Function],
                  "openInCurrentTab": false,
                  "shadow": false,
                  "to": "",
                  "type": "button",
                  "variant": "primary-small",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`renders without data 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PartyRequestFormModal />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="wh100 modal modal--light-gray-border max-width-2 mx-auto relative flex items-center justify-center bg-white drop-shadow transition-slide-up-large-long"
>
        <div
                className="wh100 m-auto flex flex-column mb3 justify-center"
        >
                <div
                        className="absolute t0 l0 w100 p3 overflow-scroll"
                >
                        <h3
                                className="callout mb3"
                        />
                </div>
                <div
                        className="absolute b0 l0 w100 p2 bg-white flex flex-row justify-end items-center"
                >
                        <Button
                                childrenWrapperClassName=""
                                className=""
                                color="madison-blue"
                                disabled={false}
                                fullWidth={false}
                                hover="shadow"
                                label="Close"
                                onClick={[Function]}
                                onMouseEnter={[Function]}
                                onMouseLeave={[Function]}
                                openInCurrentTab={false}
                                shadow={false}
                                to=""
                                type="button"
                                variant="primary-small"
                        />
                </div>
        </div>
</div>,
      "className": "overflow-scroll fixed-cover bg-white-wash flex justify-center items-center transition-fade-in",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="wh100 m-auto flex flex-column mb3 justify-center"
>
          <div
                    className="absolute t0 l0 w100 p3 overflow-scroll"
          >
                    <h3
                              className="callout mb3"
                    />
          </div>
          <div
                    className="absolute b0 l0 w100 p2 bg-white flex flex-row justify-end items-center"
          >
                    <Button
                              childrenWrapperClassName=""
                              className=""
                              color="madison-blue"
                              disabled={false}
                              fullWidth={false}
                              hover="shadow"
                              label="Close"
                              onClick={[Function]}
                              onMouseEnter={[Function]}
                              onMouseLeave={[Function]}
                              openInCurrentTab={false}
                              shadow={false}
                              to=""
                              type="button"
                              variant="primary-small"
                    />
          </div>
</div>,
        "className": "wh100 modal modal--light-gray-border max-width-2 mx-auto relative flex items-center justify-center bg-white drop-shadow transition-slide-up-large-long",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="absolute t0 l0 w100 p3 overflow-scroll"
>
              <h3
                            className="callout mb3"
              />
</div>,
            <div
              className="absolute b0 l0 w100 p2 bg-white flex flex-row justify-end items-center"
>
              <Button
                            childrenWrapperClassName=""
                            className=""
                            color="madison-blue"
                            disabled={false}
                            fullWidth={false}
                            hover="shadow"
                            label="Close"
                            onClick={[Function]}
                            onMouseEnter={[Function]}
                            onMouseLeave={[Function]}
                            openInCurrentTab={false}
                            shadow={false}
                            to=""
                            type="button"
                            variant="primary-small"
              />
</div>,
          ],
          "className": "wh100 m-auto flex flex-column mb3 justify-center",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3
                  className="callout mb3"
/>,
                undefined,
              ],
              "className": "absolute t0 l0 w100 p3 overflow-scroll",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": undefined,
                  "className": "callout mb3",
                },
                "ref": null,
                "rendered": null,
                "type": "h3",
              },
              undefined,
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Button
                childrenWrapperClassName=""
                className=""
                color="madison-blue"
                disabled={false}
                fullWidth={false}
                hover="shadow"
                label="Close"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onMouseLeave={[Function]}
                openInCurrentTab={false}
                shadow={false}
                to=""
                type="button"
                variant="primary-small"
/>,
              "className": "absolute b0 l0 w100 p2 bg-white flex flex-row justify-end items-center",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": null,
                "childrenWrapperClassName": "",
                "className": "",
                "color": "madison-blue",
                "disabled": false,
                "fullWidth": false,
                "hover": "shadow",
                "label": "Close",
                "onClick": [Function],
                "onMouseEnter": [Function],
                "onMouseLeave": [Function],
                "openInCurrentTab": false,
                "shadow": false,
                "to": "",
                "type": "button",
                "variant": "primary-small",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="wh100 modal modal--light-gray-border max-width-2 mx-auto relative flex items-center justify-center bg-white drop-shadow transition-slide-up-large-long"
>
          <div
                    className="wh100 m-auto flex flex-column mb3 justify-center"
          >
                    <div
                              className="absolute t0 l0 w100 p3 overflow-scroll"
                    >
                              <h3
                                        className="callout mb3"
                              />
                    </div>
                    <div
                              className="absolute b0 l0 w100 p2 bg-white flex flex-row justify-end items-center"
                    >
                              <Button
                                        childrenWrapperClassName=""
                                        className=""
                                        color="madison-blue"
                                        disabled={false}
                                        fullWidth={false}
                                        hover="shadow"
                                        label="Close"
                                        onClick={[Function]}
                                        onMouseEnter={[Function]}
                                        onMouseLeave={[Function]}
                                        openInCurrentTab={false}
                                        shadow={false}
                                        to=""
                                        type="button"
                                        variant="primary-small"
                              />
                    </div>
          </div>
</div>,
        "className": "overflow-scroll fixed-cover bg-white-wash flex justify-center items-center transition-fade-in",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="wh100 m-auto flex flex-column mb3 justify-center"
>
            <div
                        className="absolute t0 l0 w100 p3 overflow-scroll"
            >
                        <h3
                                    className="callout mb3"
                        />
            </div>
            <div
                        className="absolute b0 l0 w100 p2 bg-white flex flex-row justify-end items-center"
            >
                        <Button
                                    childrenWrapperClassName=""
                                    className=""
                                    color="madison-blue"
                                    disabled={false}
                                    fullWidth={false}
                                    hover="shadow"
                                    label="Close"
                                    onClick={[Function]}
                                    onMouseEnter={[Function]}
                                    onMouseLeave={[Function]}
                                    openInCurrentTab={false}
                                    shadow={false}
                                    to=""
                                    type="button"
                                    variant="primary-small"
                        />
            </div>
</div>,
          "className": "wh100 modal modal--light-gray-border max-width-2 mx-auto relative flex items-center justify-center bg-white drop-shadow transition-slide-up-large-long",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="absolute t0 l0 w100 p3 overflow-scroll"
>
                <h3
                                className="callout mb3"
                />
</div>,
              <div
                className="absolute b0 l0 w100 p2 bg-white flex flex-row justify-end items-center"
>
                <Button
                                childrenWrapperClassName=""
                                className=""
                                color="madison-blue"
                                disabled={false}
                                fullWidth={false}
                                hover="shadow"
                                label="Close"
                                onClick={[Function]}
                                onMouseEnter={[Function]}
                                onMouseLeave={[Function]}
                                openInCurrentTab={false}
                                shadow={false}
                                to=""
                                type="button"
                                variant="primary-small"
                />
</div>,
            ],
            "className": "wh100 m-auto flex flex-column mb3 justify-center",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3
                    className="callout mb3"
/>,
                  undefined,
                ],
                "className": "absolute t0 l0 w100 p3 overflow-scroll",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": undefined,
                    "className": "callout mb3",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "h3",
                },
                undefined,
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Button
                  childrenWrapperClassName=""
                  className=""
                  color="madison-blue"
                  disabled={false}
                  fullWidth={false}
                  hover="shadow"
                  label="Close"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  openInCurrentTab={false}
                  shadow={false}
                  to=""
                  type="button"
                  variant="primary-small"
/>,
                "className": "absolute b0 l0 w100 p2 bg-white flex flex-row justify-end items-center",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": null,
                  "childrenWrapperClassName": "",
                  "className": "",
                  "color": "madison-blue",
                  "disabled": false,
                  "fullWidth": false,
                  "hover": "shadow",
                  "label": "Close",
                  "onClick": [Function],
                  "onMouseEnter": [Function],
                  "onMouseLeave": [Function],
                  "openInCurrentTab": false,
                  "shadow": false,
                  "to": "",
                  "type": "button",
                  "variant": "primary-small",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
